control_systems: # Name of Module
  ros__parameters:
    update_rate: 1000.0 #[ms]

    pure_pursuit_parameters:
      wheelbase: 4.0 #[m] //TODO - Change
      lookahead_distance_scaling_factor: 0.1 

    pid_parameters:
      steering_controller_params: [
        10.0, #p_gain
        0.002, #i_gain
        0.01, #d_gain
        
        1.0 # output_lim, absolute value limit of output
      ]

      velocity_controller_params: [
        1.0, #p_gain
        0.0, #i_gain
        0.0, #d_gain
        
        1.0 # output_lim, absolute value limit of output
      ]


